frontend/src/app/(routes)/multisig/components/DialogCreateMultisig.tsx

```diff
@@ -0,0 +1,531 @@
+import {
+  generateMultisigAccount,
+  isValidPubKey,
+} from '@/txns/multisig/multisig';
+import { getAuthToken } from '@/utils/localStorage';
+import { createAccount } from '@/store/features/multisig/multisigSlice';
+import { RootState } from '@/store/store';
+import { ADDRESS_NOT_FOUND } from '@/utils/constants';
+import { createMultisigTextFieldStyles } from '../styles';
+
+interface DialogCreateMultisigProps {
+  open: boolean;
+  onClose: () => void;
+  addressPrefix: string;
+  chainID: string;
+  address: string;
+  pubKey: string;
+  baseURL: string;
+}
+
+interface InputTextComponentProps {
+  index: number;
+  field: PubKeyFields;
+  handleRemoveValue: (index: number) => void;
+  handleChangeValue: (
+    index: number,
+    e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => void;
+}
+
+interface PubKeyFields {
+  name: string;
+  value: string;
+  label: string;
+  placeHolder: string;
+  required: boolean;
+  disabled: boolean;
+  pubKey: string;
+  address: string;
+  isPubKey: boolean;
+  error: string;
+}
+
+const MAX_PUB_KEYS = 7;
+
+const InputTextComponent: React.FC<InputTextComponentProps> = (props) => {
+  const { field, index, handleChangeValue, handleRemoveValue } = props;
+  return (
+    <>
+      <TextField
+        className="bg-[#FFFFFF0D] rounded-2xl"
+        onChange={(e) => handleChangeValue(index, e)}
+        name={field.isPubKey ? 'pubKey' : 'address'}
+        value={field.isPubKey ? field.pubKey : field.address}
+        required={field?.required}
+        placeholder={field.isPubKey ? 'Public Key (Secp256k1)' : 'Address'}
+        sx={createMultisigTextFieldStyles}
+        fullWidth
+        disabled={field.disabled}
+        InputProps={{
+          endAdornment:
+            index !== 0 ? (
+              <InputAdornment
+                onClick={() =>
+                  !field.disabled
+                    ? handleRemoveValue(index)
+                    : alert('cannot self remove')
+                }
+                position="end"
+                sx={{
+                  '&:hover': {
+                    cursor: 'pointer',
+                  },
+                }}
+              >
+                <DeleteIcon sx={{ color: deepPurple[200] }} />
+              </InputAdornment>
+            ) : null,
+          sx: {
+            input: {
+              color: 'white',
+              fontSize: '14px',
+              padding: 2,
+            },
+          },
+        }}
+      />
+      <div className="address-pubkey-field-error">
+        {field.error.length ? field.error : ''}
+      </div>
+    </>
+  );
+};
+
+const getPubkey = async (address: string, baseURL: string) => {
+  try {
+    const { status, data } = await Axios.get(
+      `${cleanURL(baseURL)}/cosmos/auth/v1beta1/accounts/${address}`
+    );
+
+    if (status === 200) {
+      return data.account.pub_key.key || '';
+    } else {
+      return '';
+    }
+  } catch (error) {
+    console.log(error);
+    return '';
+  }
```

Comment:
If pubkey nor found the error is being displayed in the createMultisigForm so no need to dispatch error.
Since we are in dev mode I am use console.log(error) in some places.